### PR TITLE
"Put Mob Into Marked Object" Admin Verb

### PR DESCRIPTION
**Pull request by pitaden**
## About The Pull Request

Adds an admin verb under Fun that puts a mob of your choosing inside of your currently marked object.

## Why It's Good For The Game

This verb would be fun for admin gimmicks and saves a lot of time compared to manually adding the required values through view variables. Also, it lets us banish kilgor into a butt.

## Changelog
:cl:
admin: Added the Put Mob Into Marked verb, under Fun
/:cl:

### CODE REVIEW

**Comment by nednaZ** [comment]
This would probably be best in the Secrets window, since the list of verbs in the admin tabs are incredibly long

**Comment by Merlin1230** [comment]
Could there also be a verb for getting people out?

**Comment by Kapu1178** [comment]
forceMove() already exists for this.

File: code/modules/admin/verbs/put_mob_into_marked.dm [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+	if(!t)
+		return alert(usr, "You need to mark an object first!", null, null, null, null)
+	if(!t.loc)
+		return alert(usr, "The object you marked ([t]) doesn't have a location!", null, null, null, null)
+
+	t.contents += target_mob
+	t.vis_contents += target_mob
+
+	var/obj/cur = t
```
Comment (Kapu1178) [inline code/modules/admin/verbs/put_mob_into_marked.dm:21]: why is this typecasted to obj

**Comment by nednaZ** [comment]
> forceMove() already exists for this.

ForceMove() will deafen the target unless they are also added to the recursive_contents_hearing_sensitive list, so it isn't an existing solution.

**Comment by Kapu1178** [comment]
> > forceMove() already exists for this.
> 
> ForceMove() will deafen the target unless they are also added to the recursive_contents_hearing_sensitive list, so it isn't an existing solution.

Unless donk broke something, no it wont.
image

**Comment by Kapu1178** [comment]
I looked into it, and calling forceMove() will call Entered() on whatever the destination ATOM is, which is responsible for updating contents. 
image

Doing a raw `contents += reference` is incredibly dangerous and will break several other admin tools, and can cause problems with `GLOB` tracking of client locations.

File: code/modules/admin/verbs/put_mob_into_marked.dm (hunk 1 above)
Comment (pitaden) [reply code/modules/admin/verbs/put_mob_into_marked.dm:21]: I don't know either, which is why the typecast was removed in an as-of-yet unpushed version

**Comment by pitaden** [comment]
Haven't had been able to work on this PR in a while, but I'll start implementing these changes when I get a chance! Had no idea `contents += ref` could break things or that forceMove() and lazylists existed

**Comment by pitaden** [comment]
Gonna go ahead and close this, since I accidentally deleted my latest version and I've lost interest in finishing it

**Pull request closed without merging**